content/en/content-management/build-options.md

```diff
@@ -25,33 +25,30 @@ _build:
 
 If `always`, the page will be treated as a published page, holding its dedicated output files (`index.html`, etc...) and permalink.
 
-We extended this property from a boolean to an enum in Hugo 0.76.0. Valid values are:
-
-never
-: The page will not be included in any page collection.
+Valid values are:
 
-always (default)
-: The page will be rendered to disk and get a `RelPermalink` etc.
+  - `never`
+    : The page will not be included in any page collection.
+  - `always (default)`
+    : The page will be rendered to disk and get a `RelPermalink` etc.
+  - `link`
+    : The page will be not be rendered to disk, but will get a `RelPermalink`.
 
-link
-: The page will be not be rendered to disk, but will get a `RelPermalink`.
+Note that we extended this property from a boolean to an enum in Hugo 0.76.0.
 
 #### list
 
-Note that we extended this property from a boolean to an enum in Hugo 0.68.0.
-
 Valid values are:
+  
+  - `never`
+    : The page will not be included in any page collection.
+  - `always (default)`
+    : The page will be included in all page collections, e.g. `site.RegularPages`, `$page.Pages`.
+  - `local`
+    : The page will be included in any _local_ page collection, e.g. `$page.RegularPages`, `$page.Pages`. One use case for this would be to create fully navigable, but headless content sections.
```

Comment:
```suggestion
    : The page will be included in any _local_ page collection, e.g. `$page.RegularPages`, `.Pages`. One use case for this would be to create fully navigable, but headless content sections.
```

Reply:
Changed as requested.